From 16fee3620f5eb518f60bfff5a43cd6fd4d899fa1 Mon Sep 17 00:00:00 2001
From: Abel Vesa <abel.vesa@nxp.com>
Date: Wed, 17 Oct 2018 15:04:36 +0300
Subject: [PATCH 4887/5242] MLK-19966 arm64: imx8mq: Fix assigned clocks order
 in rm67191 dtsi

commit  5e88a7379af0c0ba5555b7350daeb775bb4871d0 from
https://source.codeaurora.org/external/imx/linux-imx.git

This fixes the wrongly ordering of assigned clocks for dcss.

Fixes: 4db22d33851f8401 ("clk: imx8mq: Switch to newly added composite-8m clock")
Signed-off-by: Abel Vesa <abel.vesa@nxp.com>
Reported-by: Mirela Rabulea <mirela.rabulea@nxp.com>
Tested-by: Mirela Rabulea <mirela.rabulea@nxp.com>
Reviewed-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dtsi
index 969be82..ef62676 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dtsi
@@ -36,8 +36,8 @@
 
 	assigned-clocks = <&clk IMX8MQ_CLK_DC_PIXEL>,
 			  <&clk IMX8MQ_CLK_DISP_AXI>,
-			  <&clk IMX8MQ_VIDEO_PLL1_REF_SEL>,
 			  <&clk IMX8MQ_CLK_DISP_RTRM>,
+			  <&clk IMX8MQ_VIDEO_PLL1_REF_SEL>,
 			  <&clk IMX8MQ_VIDEO_PLL1>;
 	assigned-clock-parents = <&clk IMX8MQ_VIDEO_PLL1_OUT>,
 				 <&clk IMX8MQ_SYS1_PLL_800M>,
-- 
1.7.9.5

